Makefile

```
@@ -70,7 +70,7 @@ $(eval $(call makemock, internal/apiserver,        IServer,            apiserver
 $(eval $(call makemock, internal/metrics,          Manager,            metricsmocks))
 
 firefly-nocgo: ${GOFILES}
-		CGO_ENABLED=0 $(VGO) build -o ${BINARY_NAME}-nocgo -ldflags "-X main.buildDate=`date -u +\"%Y-%m-%dT%H:%M:%SZ\"` -X main.buildVersion=$(BUILD_VERSION)" -tags=prod -tags=prod -v
+		CGO_ENABLED=0 $(VGO) build -o ${BINARY_NAME}-nocgo -ldflags "-X main.buildDate=`date -u +\"%Y-%m-%dT%H:%M:%SZ\"` -X main.buildVersion=$(BUILD_VERSION) -X 'github.com/hyperledger/firefly/internal/version.Date=$(DATE)' -X 'github.com/hyperledger/firefly/version.Commit=${GITREF}" -tags=prod -tags=prod -v
```

```suggestion
		CGO_ENABLED=0 $(VGO) build -o ${BINARY_NAME}-nocgo -ldflags "-X main.buildDate=`date -u +\"%Y-%m-%dT%H:%M:%SZ\"` -X main.buildVersion=$(BUILD_VERSION) -X 'github.com/hyperledger/firefly/internal/version.Date=$(DATE)' -X 'github.com/hyperledger/firefly/version.Commit=${GITREF}'" -tags=prod -tags=prod -v
```

Looks like you're missing a `'` near the end of this line. That could the cause of the build error you're seeing.

---

We'll also want to add a flag for setting the `github.com/hyperledger/firefly/internal/version.Version` to `$(BUILD_VERSION)` as well.

---

`DATE` will need to be set at the top of the Makefile too, something like:
```make
DATE := $(shell date -u +"%Y-%m-%dT%H:%M:%SZ")
```